src/common/values.ts

```diff
@@ -55,12 +55,17 @@ let hasSops = false
  */
 const writeValuesToFile = async (
   targetPath: string,
-  inValues: Record<string, any>,
+  inValues: Record<string, any> | undefined,
   overwrite = false,
 ): Promise<void> => {
-  const values = cloneDeep(inValues)
   const d = terminal('common:values:writeValuesToFile')
   const isSecretsFile = targetPath.includes('/secrets.') && hasSops
+  if (isEmpty(inValues) && isSecretsFile) {
```

Comment:
I am wondering if it is better to perform this check after calling `removeBlankAttributes`

Reply:
good idea